F/FLINT/bundled/patches/Fix-fmpz_mat_minpoly_modular-with-large-coefficients.patch

```diff
@@ -0,0 +1,162 @@
+From f8ef228807ba245538492eaf7abc47e804154eef Mon Sep 17 00:00:00 2001
```

Comment:
Have you checked locally that this patch cleanly applies to FLINT v3.2.x, and that it still passes the tests? Iirc, there have been some major changes considering matrices on the FLINT side between 3.2.x and master, so it would be good to check that